lib/undercover/plugin.rb

```diff
@@ -24,13 +24,22 @@ class DangerUndercover < Plugin
     # If there are reports then it shows the report as a warning in danger.
     # @return  [void]
     #
-    def report(undercover_path = DEFAULT_PATH, sticky: true)
-      return fail('Undercover: coverage report cannot be found.') unless valid_file? undercover_path
+    def report(undercover_path = DEFAULT_PATH, sticky: true, in_line: false, fail_0_coverage: false)
+      return warn('Undercover: coverage report cannot be found.') unless valid_file? undercover_path
 
       report = File.open(undercover_path).read.force_encoding('UTF-8')
 
       if report.match(/some methods have no test coverage/)
-        warn(report, sticky: sticky)
+        return warn(report, sticky: sticky) unless in_line
+
+        report.each_line do |line|
+          next unless line.strip.start_with?("loc:")
+
+          _, filename, from_line, to_line = *line.match(/loc:\s([^:]*):(\d+):(\d+)/)
```

Comment:
Filename probably shouldn't be empty too?
```suggestion
          _, filename, from_line, to_line = *line.match(/loc:\s([^:]+):(\d+):(\d+)/)
```

Also please mind, that colon is a valid filename character in linux. It's not a big deal for us, but should be considered if contributing to mainstream.

Reply:
I don't think this will do any arm in adding :)